rbac/server/api/packs.py

```diff
@@ -140,22 +141,22 @@
     """
     txn_key, txn_user_id = await utils.get_transactor_key(request)
 
-    pack = await packs_query.check_pack_by_pack_id(request.app.config.DB_CONN, pack_id)
+    pack = await packs_query.get_pack_by_pack_id(request.app.config.DB_CONN, pack_id)
     if not pack:
         raise ApiBadRequest(
             "Error: Pack does not currently exist or has already been deleted."
         )
     owners = await packs_query.get_pack_owners_by_id(
         request.app.config.DB_CONN, pack_id
     )
-    if txn_user_id not in owners or not utils.check_admin_status(txn_user_id):
+    if txn_user_id not in owners[0] or not utils.check_admin_status(txn_user_id):
```

Comment:
I know we haven't seen any instances of this yet, but what happens if there are multiple owners for a pack? get_pack_owners_by_id() will return a a list of lists containing the next_ids of the pack owners. 

Maybe we can utilize the fetch_relationship() function in rbac/server/db/relationships_query.py? I believe fetch_relationships will just return a list of next_ids from the pack_owners table for a specific pack if you call fetch_relationships("pack_owners", "pack_id", pack_id)